renpy/bootstrap.py

```diff
@@ -306,6 +312,14 @@ def bootstrap(renpy_base):
             except Exception as e:
                 renpy.error.report_exception(e)
 
+        if renpy.emscripten:
+            import emscripten
+            if exit_status == 0:
+                emscripten.run_script('Module.print("The game quits. You may close the tab.")')
```

Comment:
Maybe these should allow translations, using something like this:
```diff
-                 emscripten.run_script('Module.print("The game quits. You may close the tab.")')
+                 emscripten.run_script('Module.print("{}")'.format(__("The game quits. You may close the tab.")))
```
That way the messages, and them only, could be translated through common.rpym without risk of typos in the emscripten call.